src/combobox/combobox.stories.ts

```diff
@@ -15,11 +15,13 @@ storiesOf("Combobox", module)
 	.addDecorator(withKnobs)
 	.add("Basic", () => ({
 		template: `
-			<ibm-combo-box [items]="items" (selected)="selected($event)">
+			<ibm-combo-box [label]="label" [helperText]="helperText" [items]="items" (selected)="selected($event)">
```

Comment:
These attrs should probably get split onto newlines (same for the rest of the stories)